smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -83,24 +85,25 @@ function displayReaderList(readers) {
   for (let reader of readers) {
     GSC.Logging.checkWithLogger(logger, readersListElement !== null);
     goog.asserts.assert(readersListElement);
-
+    
     var indicatorClasses = 'reader-state-indicator reader-state-indicator-' +
-                           reader['status'];
+    reader['status'];
     if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.SUCCESS &&
-        reader['isCardPresent']) {
+    reader['isCardPresent']) {
       indicatorClasses = 'reader-card-present-indicator';
     }
     var indicator = goog.dom.createDom('span', indicatorClasses);
-
+    
```

Comment:
nit: I think these changes should be reversed - lines should not end with whitespace, and if a statement is not completed in one line the following line should start with at least 4 more space characters than the line before it - often as much that it starts at the same height as the statement that it continues (so the way it was before looks good)

Reply:
not exactly sure i understand you correctly, i didn't change any code in this part.
are you referring to the extra spaces on line 88 and 96?